zmap/zgrab2
# preserve protocols order, specified in ini file for multiple mode

Update zflags module to new version, that preserves ini file sections order, so protocols will be probed in order, specified in ini config file for multiple mode.

It is especially useful when `break-on-success` flag is used to specify protocols probe order and try more popular protocols first.

## How to Test

Set at least 2 different protocols in multiple mode configuration file with the same port, e.g. ssh and http:

```ini
...
[ssh]
port=80
[http]
port=80
```

Run multiple mode with `break-on-success` option on host, having http server on specified port, e.g. icanhazip.com. Make sure both ssh and http were tried.

Then invert protocols order in ini file:

```ini
...
[http]
port=80
[ssh]
port=80
```

and run multiple mode with `break-on-success` option again. Make sure only http but not ssh was tried.
